### PR TITLE
tests: update traces

### PR DESCRIPTION
**Pull request by anabwan**

### CODE REVIEW

File: rl_coach/tests/trace_tests.py [hunk 1]
```diff
@@ -223,6 +284,9 @@ def main():
                 test_count += 1
                 continue
 
+            # if args.endpoint and args.bucket:
```
Comment (galnov) [inline rl_coach/tests/trace_tests.py]: Any reason to keep these lines?

Comment (anabwan) [reply rl_coach/tests/trace_tests.py]: fixed

**Comment by shadiendrawis** [comment]
I think there's no reason to save the traces in the repo, instead, both the old and the newly generated traces can live on s3, which will make it easier to handle as they are in the same place and it will not require us to push into git every time we want to update the traces.

**Comment by anabwan** [comment]
@shadiendrawis @galleibo-intel @galnov I've changed how Traces are using the files... it's working on S3 now.
please review again.

**Pull request merged**